features/accelerometer.yml

```
@@ -1,6 +1,6 @@
 name: Accelerometer
-description: The `Accelerometer` and `LinearAccelerationSensor` APIs read the acceleration applied to a device in three dimensions, either including the effect of gravity or not, respectively.
-spec: https://w3c.github.io/accelerometer/#accelerometer-interface
+description: The `Accelerometer`, `LinearAccelerationSensor` and `GravitySensor` APIs read the acceleration applied to a device in three dimensions, either including the effect of gravity, without its effect, or only its effect, respectively.
+spec: https://w3c.github.io/accelerometer/
 group: sensors
 status:
   compute_from: api.Accelerometer.Accelerometer
```

It seems like we might want to change this single key to an array, now that we're folding the GravitySensor API in here.

```suggestion
  compute_from:
    - api.Accelerometer.Accelerometer
    - api.GravitySensor
```

This will push out the chrome versions from 67 to 91, and the edge version from 79 to 91, which I think is fine. The feature wasn't baseline before, and won't be after either because other browsers don't support it. Advancing those version numbers won't change that fact.